mne/viz/topo.py

```diff
@@ -236,8 +236,8 @@ def _plot_topo_onpick(event, show_func):
 def _compute_scalings(bn, xlim, ylim):
     """Compute scale factors for a unified plot"""
     pos = bn.pos
-    bn.x_s = pos[2] / (xlim[0] - xlim[1])
-    bn.x_t = pos[0] - bn.x_s * xlim[1]
+    bn.x_s = pos[2] / (xlim[1] - xlim[0])
+    bn.x_t = pos[0] - bn.x_s * xlim[0]
```

Comment:
I think this was making the unified plots mirror images of the time series. Got me really confused for a while.